db/migrate/20231025071251_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[7.1]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo, default: "image_link", null: false
+      t.text :bio
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] You've done a great job so far creating the migration tables but you seemed to have missed a minor detail in the users table. As you can see in the ERD below the `users` table has a column for `posts_counter` so kindly make sure to add it to your migration file above.
- Users table ERD
  ![image](https://github.com/Kccendana/blog/assets/104968236/b55125bc-ae1b-43f3-b6d7-373389c41ddb)